app/views/components/_nav_drawer.erb

```diff
@@ -1,34 +0,0 @@
-<drawer class="usa-width-one-fourth drawer" aria-hidden="true">
```

Comment:
I think `aria-hidden` is why this was not showing...but in any case, we didn't need it!